src/POGOProtos/Map/Pokemon/WildPokemon.proto

```diff
@@ -10,5 +10,5 @@ message WildPokemon {
 	double longitude = 4;
 	string spawn_point_id = 5;
 	.POGOProtos.Data.PokemonData pokemon_data = 7;
-	int32 time_till_hidden_ms = 11;
+	int64 time_till_hidden_ms = 11;
```

Comment:
Value is int32 in the game master dumps; it had previously been overflowed but has now counted down into int32 range. Probably fine to leave it the way it is, as it seems intended. Map implementations should be treating time_till_hidden_ms over 900000 or <0 as a valid 15 min spawn.